server/cliserver_test.go

```diff
@@ -34,6 +40,49 @@ func TestGetStatus(t *testing.T) {
 	defer res.Body.Close()
 	body, err := ioutil.ReadAll(res.Body)
 	req.Nil(err)
-	assert.Equal(`{"Manifests":{},"OrchestratorPool":[],"Version":"undefined","RegisteredTranscodersNumber":1,"RegisteredTranscoders":[{"Address":"TestAddress","Capacity":5}],"LocalTranscoding":false}`,
-		string(body))
+	expected := fmt.Sprintf(`{"Manifests":{},"OrchestratorPool":[],"Version":"undefined","GORuntimeVersion":"%s","GOArch":"%s","GOOS":"%s","RegisteredTranscodersNumber":1,"RegisteredTranscoders":[{"Address":"TestAddress","Capacity":5}],"LocalTranscoding":false}`,
```

Comment:
As per @j0sh 's suggestion, we should also change "GORuntimeVersion" to "GolangRuntimeVersion"

Reply:
fixed